corporate/models.py

```diff
@@ -26,7 +32,15 @@ def get_customer_by_realm(realm: Realm) -> Optional[Customer]:
 
 
 class CustomerPlan(models.Model):
+    """
+    This is for storing most of the fiddly details
+    of the customer's plan.
+    """
+
+    # Currently there can be only one plan per customer, so this
+    # is really a one-to-one relationship.
```

Comment:
A customer object can have multiple expired plans associated with it. It can indeed have only one active plan.

Reply:
Ah, I was wrong about this then - pushed an amended commit explaining this more correctly.